pkg/ccl/backupccl/testdata/backup-restore/alter-schedule/backup-options

```diff
@@ -102,3 +102,33 @@ with schedules as (show schedules) select id, command->'backup_statement' from s
 ----
 $fullID "BACKUP INTO 'nodelocal://1/example-schedule' WITH revision_history = true, detached"
 $incID "BACKUP INTO LATEST IN 'nodelocal://1/example-schedule' WITH revision_history = true, detached"
+
+exec-sql
+alter backup schedule $incID set with include_all_secondary_tenants = true
+----
+
+query-sql
+with schedules as (show schedules) select id, command->'backup_statement' from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+$fullID "BACKUP INTO 'nodelocal://1/example-schedule' WITH revision_history = true, detached, include_all_secondary_tenants = true"
+$incID "BACKUP INTO LATEST IN 'nodelocal://1/example-schedule' WITH revision_history = true, detached, include_all_secondary_tenants = true"
+
+
+exec-sql
+create schedule 'with-secondary' for backup into 'nodelocal://1/example-schedule-with-secondary' WITH include_all_secondary_tenants recurring '@daily' full backup '@weekly';
+----
+
+let $withSecondaryFullID $withSecondaryIncID
+with schedules as (show schedules) select id from schedules where label='with-secondary' order by command->>'backup_type' asc;
+----
+
+query-sql
+with schedules as (show schedules) select id, command->'backup_statement' from schedules where label='with-secondary' order by command->>'backup_type' asc;
```

Comment:
nit: can we elide the id in the output? The way our data-driven framework replaces `$` values causes a  `--rewrite` to replace all the $ values in the output with the actual job IDs. This makes `--rewrite` a little difficult to use on these `alter-schedule` tests because you have to manually go and replace those job IDs with the $ variables. There is probably a correct way to fix this, but in the 2 mins I spent looking at it I couldn't figure it out.